notebooks/community/aisamples/AIsample - Time Series Forecasting.ipynb

```diff
@@ -398,6 +398,59 @@
             "source": [
                 "display(df_p)"
             ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "## Step 4: Log and Load Model with MLFlow\n",
+                "We can now store the trained model for later use."
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "# setup mlflow\n",
+                "import mlflow\n",
+                "import trident.mlflow\n",
+                "from trident.mlflow import get_sds_url\n",
+                "import uuid\n",
+                "\n",
+                "EXPERIMENT_NAME = \"aisample-prophet\"\n",
+                "\n",
+                "mlflow.set_tracking_uri(get_sds_url())\n",
+                "mlflow.set_registry_uri(get_sds_url())\n",
+                "mlflow.set_experiment(EXPERIMENT_NAME)"
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "# log the model and parameters\n",
+                "model_name = f\"{EXPERIMENT_NAME}-{uuid.uuid4()}\"\n",
```

Comment:
Can we remove this uuid as run_id is already random?

Reply:
You are correct, random name is useless. Model name changed